Gems/Prefab/PrefabBuilder/PrefabGroup/PrefabGroup.cpp

```diff
@@ -138,6 +138,8 @@ namespace AZ::SceneAPI::SceneData
                         ->Attribute(AZ::Edit::Attributes::NameLabelOverride, "")
                         ->Attribute(AZ::Edit::Attributes::Max, 1)
                         ->Attribute(AZ::Edit::Attributes::CategoryStyle, "display divider")
+                        // There isn't a documentation page for default prefabs under the scene settings documentation category, yet.
```

Comment:
Nit: May worth cutting a GHI for that :)

Reply:
The prefab documentation https://www.o3de.org/docs/user-guide/interactivity/prefabs/ is a stub that links to a GHI https://github.com/o3de/o3de.org/issues/962 , which I thought was still open but is closed. I've asked for an update on the GHI and default prefab info.

Reply:
We can get some useful info/links on this index page in the next few days.